installation/resources/installer.yaml

```diff
@@ -196,7 +196,7 @@ spec:
             readOnly: true
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:408cb6a6
+        image: eu.gcr.io/kyma-project/develop/installer:1.11.0-rc1
```

Comment:
it should be kyma-installer and please check folder cause i think it's not develop anymore

Reply:
you are right, thanks!